lavalink/models.py

```diff
@@ -288,8 +304,8 @@ def add(self, requester: int, track: dict, index: int = None):
             band = value[0]
             gain = value[1]
 
-            if -1 > value[0] > 15:
-                continue
+            if not -1 < value[0] < 15:
+                raise InvalidBand('{} is a invalid band, must be 0-14'.format(band))
```

Comment:
Generally, user errors should be halted immediately and the issue should be made known to the user. I do see your point, but I believe this error has a more predictable behavior; you see an error so you expect execution of the function to stop. If it proceeds with execution, it might confuse the user. Additionally, the user of this library can catch these errors and perhaps forward them onto their end-users, so that if they implement EQ functionality and the end-user passes an invalid band, the message can be forwarded on to let the user know the provided input is invalid.

As for the grammar issue, `an invalid` are the correct words so this will need changing in that respect.

Also on another note, I can't remember if I mentioned it but I'm torn between whether this deserves its own exception class or if it should be an `IndexError`.

Reply:
I think just using IndexError would be better.